components/projectsView/ProjectsList.js

```diff
@@ -39,15 +39,24 @@ function applyCurrentFilters(issues, currentFilters, filterDefs) {
   const activeFilterDefs = filterDefs.filter(
     (filterDef) => currentFilters[filterDef.key]
   );
+
   return issues.filter((issue) => {
     // test each issue against all active filters
-    return activeFilterDefs.every((filterDef) => {
+    const isMatch = activeFilterDefs.every((filterDef) => {
       return filterDef.matchFunc(
         issue,
         filterDef.field,
         currentFilters[filterDef.key]
       );
     });
+
+    // if status is 'completed', filter out issues with the label "Archived Project"
+    if (currentFilters.status === "completed") {
```

Comment:
What do you think about moving this logic up into the `matchFunc` passed in the `FILTER_DEFS` config since it is specific to the status field? I could see it both ways, but it seems like `applyCurrentFilters` was written to be as general as possible.

Reply:
two questions on this:
1. Are we sure we only want to hide archived projects which are "completed" - or should we simply hide any project with the `Archived Project` label?
2. Any reason not to mange this via the Socrata query, like Frank did here: https://github.com/cityofaustin/atd-product/pull/86/commits/3eba729518dc23e34657f95e2e06fbbcb31136a3

Reply:
@mddilley Great suggestion! I have hauled that logic up into a function defined in that portion of the filter definitions. This may be moot however based on the resolution of John's queries. 

@johnclary - thank you for asking. I'll await Amenity's decision on how to refine the issue.